Vagrantfile

```diff
@@ -5,7 +5,7 @@ CENTOS_8_BOX_URL = "https://cloud.centos.org/centos/8-stream/x86_64/images/CentO
 CENTOS_9_BOX_URL = "https://cloud.centos.org/centos/9-stream/x86_64/images/CentOS-Stream-Vagrant-9-latest.x86_64.vagrant-libvirt.box"
 
 Vagrant.configure("2") do |config|
-  config.vm.box = "centos/7"
+  config.vm.box = "centos/stream9"
```

Comment:
How to deal with the box override? https://app.vagrantup.com/centos/boxes/stream9 is still empty now.

Reply:
if "cleanly" errors if there is no override, so I'd leave it as is for now